.flake8

```diff
@@ -0,0 +1,12 @@
+# A flake8 linting config adjusted for black autoformatting, defined mainly
+# based on the following documentation:
+#
+# https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length
+#
+[flake8]
+max-line-length = 88
```

Comment:
I usually steal from https://github.com/jupyterhub/jupyterhub/blob/main/.flake8, but stricter (see https://github.com/2i2c-org/pilot-hubs/blob/master/.flake8). In particular, I don't want flake8 to impose a line length limit...

Reply:
I have stolen that often as well in the past, but after working with @mathbunnyru in jupyter/docker-stacks, I learned that a lot was covered automatically by using black's autoformatting and that it therefore could be reduced to something like this.